system/jlib/jtrace.cpp

```diff
@@ -1178,13 +1239,15 @@ class CNullSpan final : public CInterfaceOf<ISpan>
 
     virtual ISpan * createClientSpan(const char * name, const SpanTimeStamp * spanStartTimeStamp = nullptr) override { return getNullSpan(); }
     virtual ISpan * createInternalSpan(const char * name, const SpanTimeStamp * spanStartTimeStamp = nullptr) override { return getNullSpan(); }
+    virtual ISpan * createConditionalInternalSpan(const char * name, stat_type thresholdMs) override { return getNullSpan(); }
+    virtual ISpan * createConditionalClientSpan(const char * name, stat_type thresholdMs) override { return getNullSpan(); }
 
 private:
     CNullSpan(const CNullSpan&) = delete;
     CNullSpan& operator=(const CNullSpan&) = delete;
 };
 
-
+enum class TernaryBool { Uninitialized, True, False };
```

Comment:
might not be necessary, but this seems to be a clean way to determine if a flag has been initialized (wouldn't be surprised if we have something in place)